### ISSUE TITLE
0.18 Breaking change - apiName instead of apiGroup showing in template

### ISSUE THREAD

**Issue by tomgrossman** [apidoc/apidoc#821]
In version 0.18.0, you released this "fix":
**Fix apiGroup showing instead of apiName in template**
This is a breaking change, not a minor version fix. Since it changes the presentation template completely.

In addition, until now the `@apiName` was used only in the `apidoc.json` for the order of the routes. Now when it is presented, it looks like it supposed to be instead of the `@api title` which is optional, but without it, it won't appear in the navigation bar.
So I have to use both `@apiName` && `title` which are the same, so I have duplication in the template.

In short - breaking change and even when I try to fix the change, it's not usable.

# Information
- Version of apidoc: 0.19.0
- Operating system (e.g. GNU+Linux/MacOS/Windows): Linux
- Browser (e.g. firefox, chrome): chrome

**Comment by NicolasCARPi** [comment]
Note: this change is related to https://github.com/apidoc/apidoc/issues/790

> when I try to fix the change, it's not usable.

The change is very simple: https://github.com/apidoc/apidoc/commit/5de39e839e69365937f7ab13ff5cfb9e6fbd8c6e what do you mean you can't revert it?

As for which change should be kept, I'm not sure tbh.

**Comment by tomgrossman** [comment]
@NicolasCARPi I mean that until this change, my title was used instead of the name. And the name was used only for the order of the functions in the apidoc.json file.
Now, I MUST use both. Because the title is mandatory for the navigation bar, and the name is used for the ordering.
The issue is that they are exactly the same for me. **title is the name and the name is the title**.
IMO it was much better to show in the template `$group - $title`.

Anyway, even if you don't agree with me, this is not a bug fix. It's a major version since it's a change in the API.

**Comment by NicolasCARPi** [comment]
If you look into the `example` folder, the apiName will be something like GetUser and the api title will be "Get info on a user". I think that's how it was intended to be used, with the name being short and used in the url and stuff, and the title being in complete english.

> Anyway, even if you don't agree with me

I don't agree or disagree. As I said, I'm not sure if the change was good or not. At the time, it seemed good. Would love to have other users input on this.

>  this is not a bug fix. It's a major version

That doesn't make sense for a 0.x version software, which is apidoc. Before version 1 it means that the api can break, it wouldn't make sense to go to 1.x for this change. But we can both agree that this change should have been advertised as breaking in the changelog. But given that you're the only one complaining about it, maybe for other users it wasn't an issue (for me it didn't change a thing). So I'd say that it is a breaking change for you because of the way you annotated the code, right?

Also, please consider that I have not written this software, I'm just giving out some of my free time to the community to maintain and try to improve it, and I'm trying my best.

**Comment by tomgrossman** [comment]
@NicolasCARPi I'm not blaming you of course :)
Just saying that this is a breaking change, and in order to use it properly I will have to change all of my documentation.
So for now, I have to stay with version 0.17.

**Comment by tomgrossman** [comment]
@NicolasCARPi Found another example of a public API using the group as the prefix of the titles:
https://api.passivetotal.org/index.html

They will also face the same bug if they upgrade...

**Comment by NicolasCARPi** [comment]
Can you confirm that reverting the commit mentioned above entirely fixes the issue for you?

**Comment by tomgrossman** [comment]
@NicolasCARPi yes, it fixes.
